docs/api/covidcast-signals/doctor-visits.md

```diff
@@ -35,13 +35,18 @@ several days after they occur, these signals are typically available with
 several days of lag. This means that estimates for a specific day are only
 available several days later.
 
+Note that we expect estimates available for the most recent X days to be substantially revised due to later data revisions. 
+
+As insurnace claims are available at a significant and variable latency, the signal experiences heavy backfill with data delayed for a couple of weeks. Visit [Doctor's Visits](https://delphi.cmu.edu/blog/2020/11/05/a-syndromic-covid-19-indicator-based-on-insurance-claims-of-outpatient-visits/#backfill) for more information on backfill.
```

Comment:
The "Doctor's Visits" text is filler, I was unsure what language to use to refer to the blog post.

Reply:
```suggestion
As insurance claims are available at a significant and variable latency, the signal experiences heavy backfill with data delayed for a couple of weeks. See our [blog post](https://delphi.cmu.edu/blog/2020/11/05/a-syndromic-covid-19-indicator-based-on-insurance-claims-of-outpatient-visits/#backfill) for more information on backfill.
```